lib/kafka/consumer.rb

```diff
@@ -572,10 +592,35 @@ def clear_current_offsets(excluding: {})
       end
     end
 
+    def scan_for_subscribing
+      @subscribed_topics.keys.each do |topic_or_regex|
+        default_offset = @subscribed_topics[topic_or_regex][:default_offset]
+        start_from_beginning = @subscribed_topics[topic_or_regex][:start_from_beginning]
+        max_bytes_per_partition = @subscribed_topics[topic_or_regex][:max_bytes_per_partition]
```

Comment:
Would it not make sense to use `each` here instead, e.g.

```ruby
@subscribed_topics.each do |topic_or_regex, topic_config|
  default_offset = topic_config.fetch(:default_offset) # also I prefer `fetch` because it crashes explicitly
  ...
end
```

Reply:
Done